### ISSUE TITLE
Missing rule

### ISSUE THREAD

**Issue by frack113** [SigmaHQ/sigma#1941]
HI,
@austinsonger reported me that "sysmon_dns_over_https_enabled.yml" is missing.
I have try 
```bash
 git log --all --full-history "*sysmon_dns_over_https_enabled.yml"
```
But can find when the rule was remove after #1724 
It not a sigma bug but can someone good with git help me ?
Thanks

**Comment by austinsonger** [comment]
@frack113  I can always just do another pull request. But it would be nice to figure out what happened.

**Comment by frack113** [comment]
If you want, I have open a issue to get some help in my search.

**Comment by frack113** [comment]
Ok ,
find
```bash
git log --diff-filter=D --summary --reverse | grep delete | grep sysmon_dns_over_https_enabled
 delete mode 100644 rules/windows/registry_event/sysmon_dns_over_https_enabled.yml
 delete mode 100644 rules/windows/registry_event/sysmon_dns_over_https_enabled.yml
```
```bash
git log --summary --reverse -- rules/windows/registry_event/sysmon_dns_over_https_enabled.yml
commit a4b78ef4f064b93b816b4aabad9ea87d10a343f5
Author: Austin Songer <austin@songer.pro>
Date:   Thu Jul 22 21:48:28 2021 -0500

    Delete sysmon_dns_over_https_enabled.yml

 delete mode 100644 rules/windows/registry_event/sysmon_dns_over_https_enabled.yml
(END)
```
finally https://github.com/SigmaHQ/sigma/commit/a4b78ef4f064b93b816b4aabad9ea87d10a343f5
